src/dwarf.cpp

```diff
@@ -356,14 +356,16 @@ std::size_t fatal_attribute_hash(const attribute_sequence& attributes) {
 
     std::size_t h{0};
 
-    std::for_each_n(&names[0], count, [&](const auto& name) {
+    for (std::size_t i{0}; i < count; ++i) {
```

Comment:
Yeah yeah, no raw loops; but this is easier to debug 🤷

Reply:
There are so many cases where "no raw loops" is just too academic. No complaints from me.